### PR TITLE
Enhanced logging of profits to get a deeper understanding/control of …

### PR DESCRIPTION
**Pull request by sydekumf**
…the gains and profits of this bot

I always had the struggle to understand the actual profit/gains of this bot. This is why I enhanced the logging and output of it. I need gains in percentage not in total numbers. I always want to see different types of gains, like in BTC (which would be the most neutral one, as USDT pair changes) and on different time scales, like 1 day or 7 days.

Current value outputs like this:
```
Last update: 20:20:09 25/05/2021

Current coin ONT:
 - Balance: 2008.57 ONT
 - Exchange rate purchased: 1.1723 USDT/ONT 
 - Exchange rate now:       1.0771 USD/ONT
 - Change in value: -8.12 %'
 - Value in USD: 2163.43 USD
 - Value in BTC: 0.05746519 BTC

1 day change BTC: -4.7 %
7 days change BTC: 11.11 %
```

Progress outputs like this:
```
ONT
 - Amount: 2008.57 ONT
 - Price: 2354.65 USD
 - Change: 148.86 ONT 8.0 % in the last 4 days, 7 hours
 - Trade datetime: 00:10:02 25/05/2021
```

### CODE REVIEW

File: btb_manager_telegram/buttons.py [hunk 1]
```diff
@@ -89,14 +115,13 @@ def current_value():
                     f"\nLast update: `{last_update.strftime('%H:%M:%S %d/%m/%Y')}`\n\n"
                     f"*Current coin {current_coin}:*\n"
                     f"\t\- Balance: `{format_float(balance)}` *{current_coin}*\n"
-                    f"\t\- Current coin exchange rate: `{format_float(usd_price)}` *USD*/*{current_coin}*\n"
+                    f"\t\- Exchange rate purchased: `{format_float(buy_price / alt_amount)}` *{bridge}*/*{current_coin}* \n"
+                    f"\t\- Exchange rate now:           `{format_float(usd_price)}` *USD*/*{current_coin}*\n"
+                    f"\t\- *Change in value*: `{round((balance * usd_price - buy_price) / buy_price * 100, 2)}` *%*'\n"
```
Comment (lorcalhost) [inline btb_manager_telegram/buttons.py:120]: Typo here:
```python
f"\t\- *Change in value*: `{round((balance * usd_price - buy_price) / buy_price * 100, 2)}` *%*'\n"
```
Should be:
```python
f"\t\- *Change in value*: `{round((balance * usd_price - buy_price) / buy_price * 100, 2)}` *%*\n"
```

**Comment by lorcalhost** [comment]
Hi @sydekumf , thanks for your contribution.  
I need to check but are you sure trade history for the previous 7 days is kept in the database by default?

As both this and PR #99 are improving on the SQL queries they should probably be merged somehow.

Great job!

Please also take a look at [`contributing guidelines`](https://github.com/lorcalhost/BTB-manager-telegram/blob/main/CONTRIBUTING.md) for correct styling when opening PRs.

**Comment by sydekumf** [comment]
@lorcalhost Thanks! I fixed the typo and code formatting!

The trade history will never be deleted I think so 7 days is available :-)

**Comment by lorcalhost** [comment]
Reverted, should fix problems described in #109 and c921d63547a7a5ae651cba30faeee4d3290541bb

**Comment by sydekumf** [comment]
@lorcalhost @DmytroLitvinov Please see this PR for an even better profit calculation USD: https://github.com/lorcalhost/BTB-manager-telegram/pull/111

Somehow the BTC values seem to not be correct in the database, but I found a better way to even calculate it in USD.

**Pull request merged**